<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BnmsTab</class>
 <widget class="QMainWindow" name="BnmsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="proteinBox" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="protLabel">
         <property name="text">
          <string>Reference:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="referenceSelect">
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QtCharts::QChartView" name="view">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Distance Matrix Tools</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMarkerMenu"/>
   <addaction name="separator"/>
   <addaction name="actionShowAverage"/>
   <addaction name="actionShowQuantiles"/>
   <addaction name="separator"/>
   <addaction name="actionLogarithmic"/>
   <addaction name="actionSavePlot"/>
  </widget>
  <action name="actionSavePlot">
   <property name="icon">
    <iconset theme="camera-photo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
   <property name="toolTip">
    <string>Save the plot to SVG or PNG file</string>
   </property>
   <property name="shortcut">
    <string>Print</string>
   </property>
  </action>
  <action name="actionShowAverage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons-custom/index.qrc">
     <normaloff>:/icons/show-average.svg</normaloff>:/icons/show-average.svg</iconset>
   </property>
   <property name="text">
    <string>Average Profile</string>
   </property>
   <property name="toolTip">
    <string>Show average profile</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionShowIndividual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons-custom/index.qrc">
     <normaloff>:/icons/show-individual.svg</normaloff>:/icons/show-individual.svg</iconset>
   </property>
   <property name="text">
    <string>Individual Profiles</string>
   </property>
   <property name="toolTip">
    <string>Show individual profiles</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionLogarithmic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons-custom/index.qrc">
     <normaloff>:/icons/logspace-off.svg</normaloff>
     <normalon>:/icons/logspace-on.svg</normalon>:/icons/logspace-off.svg</iconset>
   </property>
   <property name="text">
    <string>Logarithmic</string>
   </property>
   <property name="toolTip">
    <string>Plot data on a log10 axis</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionShowQuantiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons-custom/index.qrc">
     <normaloff>:/icons/show-quantiles.svg</normaloff>:/icons/show-quantiles.svg</iconset>
   </property>
   <property name="text">
    <string>Per-dimension Quantiles</string>
   </property>
   <property name="toolTip">
    <string>Show per-dimension quantiles</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionMarkerMenu">
   <property name="icon">
    <iconset theme="draw-star"/>
   </property>
   <property name="text">
    <string>Markers</string>
   </property>
   <property name="toolTip">
    <string>Quick selection of markers</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons-custom/index.qrc"/>
 </resources>
 <connections/>
</ui>
